internal/security/bootstrapper/interfaces/command.go

```diff
@@ -22,6 +22,8 @@ const (
 	StatusCodeNoOptionSelected = 1
 	// StatusCodeExitWithError is exit code for error case
 	StatusCodeExitWithError = 2
+	// JSONContentType is the content type for JSON based body/payload
+	JSONContentType = "application/json"
```

Comment:
why not use the constant already here: https://github.com/edgexfoundry/go-mod-core-contracts/blob/b3069d58daddec4f0489f199d23d3a6acd3e16a6/clients/constants.go#L79